src/Raven.Server/Documents/Handlers/StreamCsvDocumentQueryResultWriter.cs

```diff
@@ -27,7 +27,7 @@ internal class StreamCsvDocumentQueryResultWriter : IStreamDocumentQueryResultWr
         
         public StreamCsvDocumentQueryResultWriter(HttpResponse response, Stream stream, DocumentsOperationContext context, string[] properties = null, string csvFileName = "export")
         {
-            csvFileName = $"{csvFileName}_{SystemTime.UtcNow.ToString("yyyyMMdd:HHmm", CultureInfo.InvariantCulture)}.csv";  
+            csvFileName = $"{csvFileName}_{SystemTime.UtcNow.ToString("yyyyMMdd_HHmm", CultureInfo.InvariantCulture)}.csv";  
```

Comment:
You didn't have that in the 4.1 PR?

Reply:
Nevermind, I see it was already merged

Reply:
This fix is here because I still needed to change
**from** yyyyMMdd:HHmm **to** yyyyMMdd_HHmm 
(use the underscore...)